### DISCUSSION TITLE
Reset Values like padding/margin...

### DISCUSSION THREAD

**Discussion by erikdemarco** [elementor/elementor#29009] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I'm always frustrated, when making a changes but want to revert the changes for example to padding/margin to their original widget values. It simply cant. We are human, always makes mistakes, how can simple feature like this doesnt exists?

### Describe the solution you'd like.

please make every value can be reverted to original values

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by black-eye** [comment]
Technically you can do it using a little trick - just change the unit and the values get reset. But yeah, it's completely uinintuitive and probably more of a side effect than intended behaviour. It should be more obvious.

And when you are on it - this reset would be better done as as an on/off switch, so you could return to previously used values. It would be really handy when testing things.
